### PR TITLE
Feature hw 29

### PR DESCRIPTION
**Pull request by CyberCuCuber**
Added DB support. Ansible playbook run with option `--ask-vault-pass`. @ggramal , vault pass i will write in the telegram privat chat

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -26,6 +34,45 @@ class WrongToken(Error):
...
+
+
+Base = declarative_base()
+
+class UserDB(Base):
+    __tablename__ = "authn"
+
+    client_id = Column(Integer, primary_key=True, nullable=False)
+    login = Column(String, unique=True, nullable=False)
+    password = Column(String, nullable=False)
```
Comment (ggramal) [inline main.py:49]: we should store hash not password in db

Comment (ggramal) [reply main.py:49]: So what needs to be done:
1. password -> hash in db
2. add blocked attribute to db
3. if you are working with a list of users in memory  you need to
  1. Load all users from db into memory (python list) when app starts
  2. Update users list and update db when you change users attribute (like blocked)
  3. Update users list and db when you add new users
4. Seed db - what that means is that you need to take info from you files and add this to db IF IT DOES nOT EXIST

Comment (CyberCuCuber) [reply main.py:49]: > we should store hash not password in db

The database stores the password in sha256

**Comment by CyberCuCuber** [comment]
@ggramal i did it

**Pull request merged**